src/Overlay.js

```diff
@@ -1,13 +1,26 @@
 /*eslint-disable object-shorthand, react/prop-types */
```

Comment:
they are already added with:

```
Overlay.propTypes = {
  ...Portal.propTypes,
  ...Position.propTypes,
```

re-adding them isn't very DRY and would require that we document them again. I think in this case the linting rule is just doing a bad job, I don't want to write worse code to satisfy it.